README.md

```diff
@@ -142,30 +146,30 @@ $ python evaluate_retriever.py \
     --include_title_in_passage \
     --device_ids 0 1 2 3
 # The result should be logged as follows:
-# Recall at 1: 0.4058 (1465/3610)
-# Recall at 2: 0.5191 (1874/3610)
-# Recall at 5: 0.6443 (2326/3610)
-# Recall at 10: 0.7144 (2579/3610)
-# Recall at 20: 0.7687 (2775/3610)
-# Recall at 50: 0.8233 (2972/3610)
-# Recall at 100: 0.8504 (3070/3610)
+# Recall at 1: 0.4194 (1514/3610)
+# Recall at 2: 0.5338 (1927/3610)
+# Recall at 5: 0.6560 (2368/3610)
+# Recall at 10: 0.7235 (2612/3610)
+# Recall at 20: 0.7720 (2787/3610)
+# Recall at 50: 0.8186 (2955/3610)
+# Recall at 100: 0.8501 (3069/3610)
 ```
 
 **5. Train a reader**
 
 ```sh
 $ python train_reader.py \
     --train_file <WORK_DIR>/reader_data/nq_train.jsonl \
-    --dev_file <WORK_DIR>/reader_data/nq_dev.jsonl  \
+    --val_file <WORK_DIR>/reader_data/nq_dev.jsonl \
```

Comment:
Changed the fold name `dev` to `val` to coordinate with the naming cnvention of PyTorch Lightning.